gridstatus/tests/test_interconnection_queue.py

```diff
@@ -5,6 +5,9 @@
 
 # todo implement this
 @pytest.mark.slow
-def test_get_interconnection_queue_all():
+@pytest.mark.parametrize("iso_class", gridstatus.utils.all_isos)
+def test_get_interconnection_queue_all(iso_class):
```

Comment:
we recently added `IESO` to [`all_isos`](https://github.com/kmax12/gridstatus/blob/c12055616f3da4e34ddde181469cd8d976133a8e/gridstatus/__init__.py#L31). there is an interconnection queue [here](https://www.ieso.ca/en/Sector-Participants/Connection-Process/Application-Status), but we haven't implemented it. let's just make sure this test can handle ieso not being implemented